src/main/java/at/hannibal2/skyhanni/features/fame/UpgradeReminder.kt

```diff
@@ -0,0 +1,193 @@
+package at.hannibal2.skyhanni.features.fame
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.ProfileStorageData
+import at.hannibal2.skyhanni.events.ConfigLoadEvent
+import at.hannibal2.skyhanni.events.GuiContainerEvent
+import at.hannibal2.skyhanni.events.InventoryCloseEvent
+import at.hannibal2.skyhanni.events.InventoryFullyOpenedEvent
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.events.SecondPassedEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.ItemUtils.getLore
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.RegexUtils.anyMatches
+import at.hannibal2.skyhanni.utils.RegexUtils.matchFirst
+import at.hannibal2.skyhanni.utils.RegexUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.RegexUtils.matches
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import com.google.gson.annotations.Expose
+import net.minecraft.item.ItemStack
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.days
+import kotlin.time.Duration.Companion.seconds
+
+@SkyHanniModule
+object UpgradeReminder {
+    private val patternGroup = RepoPattern.group("fame.upgrades")
+
+    private val accountUpgradePattern by patternGroup.pattern(
+        "account",
+        "§8Account Upgrade"
+    )
+    private val profileUpgradePattern by patternGroup.pattern(
+        "profile",
+        "§8Profile Upgrade"
+    )
+    private val upgradeDurationPattern by patternGroup.pattern(
+        "duration",
+        "§8Duration: (?<duration>\\d{1,3})d"
```

Comment:
is the format always in days? when not, will detection break? cant we use TimeUtils to dynamically parse the time unit?